### PR TITLE
[CTG] replace load_yamls with load_yaml

### PR DESCRIPTION
**Pull request by trdthg**
## Description

seems not been updated synchronously when the function, variable were modified

### CODE REVIEW

**Comment by UmerShahidengr** [comment]
@trdthg we would like to keep the naming convention of `load_yaml` to be consistent everywhere. Having two functions with names `load_yaml` and `load_yamls` will create massive confusion.
I can see there are two instances of `load_yamls` used which can be replaced with `load_yaml` instead of adding more callings of `load_yamls`.
My recommendation would be to removed `load_yamls` and replace it with `load_yaml` everywhere to keep everything consistent.

File: riscv-ctg/riscv_ctg/utils.py [hunk 1]
```diff
@@ -16,8 +16,14 @@
 yaml.default_flow_style = False
 yaml.allow_unicode = True
 
-def load_yaml(foo):
+def load_yaml(foo: str | list[str]):
```
Comment (trdthg) [inline riscv-ctg/riscv_ctg/utils.py]: I really want to write a type hint here to keep it clear, although the current code does not use type hints

**Comment by trdthg** [comment]
update

File: riscv-ctg/riscv_ctg/utils.py (hunk 1 above)
Comment (trdthg) [reply riscv-ctg/riscv_ctg/utils.py]: finally deleted, because `|` is not supported in python 3.6

File: riscv-ctg/riscv_ctg/utils.py [hunk 2]
```diff
@@ -60,9 +66,6 @@ def get_instr_list():
 
     return instr_lst
 
-def load_yamls(foo):
```
Comment (UmerShahidengr) [inline riscv-ctg/riscv_ctg/utils.py:63]: Thank you for removing this function

**Review by UmerShahidengr — APPROVED** [review]
LGTM

**Pull request merged**